libafl_qemu/Cargo.toml

```diff
@@ -52,6 +52,7 @@ fork = ["libafl/fork"]
 ## Build libqasan for address sanitization
 build_libgasan = []
 build_libqasan = []
+rasan = []
```

Comment:
I'd add some info comment about what this is, for future generations reading this

Reply:
Done